packages/core/src/Stores/client-store.js

```diff
@@ -515,7 +515,11 @@ export default class ClientStore extends BaseStore {
     @computed
     get residence() {
         // TODO Instead of return residence from each individual loginid, set in once in login, this is bound by user.
-        return this.accounts[this.loginid].residence;
+        if (this.accounts[this.loginid]) {
+            return this.accounts[this.loginid].residence;
+        }
+
+        return '';
```

Comment:
```suggestion
        return this.accounts[this.loginid]?.residence ?? '';
```

Reply:
Ahh, I forgot about the optional chaining we just added in. Thanks 👍 🌮